rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/edition/EditNodeSourceWindow.java

```diff
@@ -71,85 +70,27 @@ protected NodeSourceAction getNodeSourceAction() {
         return NodeSourceAction.EDIT;
     }
 
-    @Override
-    protected void populateFormValues() {
-        this.controller.fetchSupportedInfrastructuresAndPolicies(() -> fetchNodeSourceConfigurationWithCallback(),
-                                                                 this.window::hide);
-    }
-
     @Override
     protected List<FormItem> handleNonTextualPluginField(PluginDescriptor plugin, PluginDescriptor.Field pluginField) {
-        return inlineItemModificationCreator.getModificationChoiceItemsForNonTextualFields(plugin, pluginField);
-    }
-
-    @Override
-    protected void addButtonsToButtonsLayout(HLayout buttonsLayout) {
-        buttonsLayout.setMembers(this.deployNowButton, this.saveAndKeepUndeployedButton, this.cancelButton);
+        return new InlineItemModificationCreator(this).getModificationChoiceItemsForNonTextualFields(plugin,
+                                                                                                     pluginField);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)